### PR TITLE
a fix for the meraderm depending on hugged people

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
this makes so scalps can be grinded in larva jelly
fixes the bug with scalping (limiting it to ONE. but also allowing any xenomorph to be scalped)

## Why It's Good For The Game
strays further yet from capt gameplay and with the trending of less and less larvas getting meraline and dermaline became a pain

## Changelog
:cl:
tweak: scalps can now be grinded for jellyÇ
fix: xenos can only be scalped once. and every xeno can now be scalped
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -14. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by David-Stormwell** [comment]
Rename it from larva jelly to something different.

**Comment by Chaznoodles** [comment]
You can order Lemoline from Req. Just make that somewhat cheaper so advanced chemicals are still reliant on marines holding areas/killing xenos rather than doing the infiniscalp thing.

**Comment by joookschad** [comment]
> You can order Lemoline from Req. Just make that somewhat cheaper so advanced chemicals are still reliant on marines holding areas/killing xenos rather than doing the infiniscalp thing.

you could also fix scalp code so a xeno can be scalped only one

**Comment by SplinterGP** [comment]
Xenos can be infinitely scalped, so i'd ask to either make them be scalpable once or some workaround, since xenos also can salvage biomass and get rid of a corpse.

**Comment by fernandos13** [comment]
fixed

File: code/datums/elements/scalping.dm [hunk 1]
```diff
@@ -6,18 +6,27 @@
 	. = ..()
 	UnregisterSignal(source, COMSIG_ITEM_ATTACK)
 
-/datum/element/scalping/proc/on_attack(datum/source, mob/living/M, mob/living/user)
+/datum/element/scalping/proc/on_attack(datum/source, mob/living/carbon/xenomorph/M, mob/living/user)
```
Comment (SplinterGP) [inline code/datums/elements/scalping.dm:9]:
```suggestion
/datum/element/scalping/proc/on_attack(datum/source, mob/living/M, mob/living/user)
```
no need for this

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/datums/elements/scalping.dm [hunk 2]
```diff
@@ -10,14 +10,23 @@
...
 	if(!isxeno(M) || (M.stat != DEAD))
 		return NONE
+
 	if(M.a_intent == INTENT_HARM)
 		return NONE
+
+	if(HAS_TRAIT(M, TRAIT_SCALPED))
+		M.visible_message("<span class='notice'>[user] someone already took \the [M] scalp</span>")
+		return NONE
+
```
Comment (TiviPlus) [inline code/datums/elements/scalping.dm:20]: why is this a visible mesage

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/datums/elements/scalping.dm (hunk 2 above)
Comment (fernandos13) [reply code/datums/elements/scalping.dm:20]: oh i set that for everyone mistake

**Pull request closed without merging**